src/languages/en_US.json

```diff
@@ -545,7 +545,7 @@
         "read_more_button": "Read more",
         "available_networks": "Available on %{availableNetworks} networks",
         "available_network": "Available on the %{availableNetwork} network",
-        "available_networkv2": "Also available on %{availableNetwork}",
+        "available_networkv2": "Only available on %{availableNetwork}",
```

Comment:
copy change to support if the asset is only on 1 network, which previously was not possible